<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: black;
    color: #FFFFFF;
}

QLabel {
    color: #FFFFFF;
}</string>
  </property>
  <widget class="QLineEdit" name="filenameDisplay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>68</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source File Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>68</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Suggested New File Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>328</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Records:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>825</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">TableWidget {
    background-color: #F0F0F0;
    font-family: Arial;
    color: #000000;

}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>328</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Select Record to Edit:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_SelectRecord">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>322</y>
     <width>104</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>370</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>age</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>370</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>shirtSize</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>370</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>pantSize</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>370</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>shoeSize</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>isAvailable</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>370</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>notes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recordChange_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recordChange_age">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="recordChange_gender">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="recordChange_shirtSize">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="recordChange_pantSize">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="recordChange_shoeSize">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="recordChange_isAvail">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>322</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="recordChange_notes">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveRecord">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>440</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Save Record</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>60</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>448</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>New File Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
